specs/core/0_beacon-chain.md

```diff
@@ -1147,30 +1148,44 @@ def verify_bitfield(bitfield: bytes, committee_size: int) -> bool:
     return True
 ```
 
-### `verify_slashable_attestation`
+### `convert_to_indexed`
 
 ```python
-def verify_slashable_attestation(state: BeaconState, slashable_attestation: SlashableAttestation) -> bool:
+def convert_to_indexed(state: BeaconState, attestation: Attestation):
     """
-    Verify validity of ``slashable_attestation`` fields.
+    Convert an attestation to (almost) indexed-verifiable form
     """
-    if slashable_attestation.custody_bitfield != b'\x00' * len(slashable_attestation.custody_bitfield):  # [TO BE REMOVED IN PHASE 1]
-        return False
+    return IndexedAttestation(
+        validator_indices=get_attestation_participants(state, attestation.data, attestation.aggregation_bitfield),
+        data=attestation.data,
+        custody_bitfield=attestation.custody_bitfield,
```

Comment:
This `custody_bitfield` here is incorrect. We need to truncate the bitfield to be the `ceildiv8`of the size of the participants (`validator_indices`) and if we sort the `validator_indices`, the newly constructed custody_bitfield needs to reflect this sort order.

Reply:
Otherwise it:
* fails `verify_bitfield` because the length of `validator_indices` can easily be less than the length of the original committee 
* even if all validators participated the custody_bitfield is then in the incorrect order wrt `validator_indices`

Reply:
The best I've thought of so far is to modify `IndexedAttestation` so that it has `bit_0_validator_indices` and `bit_1_validator_indices`, and get rid of the custody bitfield.

Reply:
That will make `IndexedAttestation` inside of `AttesterSlashing` almost 2x in size.
Did you check out the fix I put in place?

Reply:
> That will make IndexedAttestation inside of AttesterSlashing almost 2x in size.

Why would it? The total size of the two lists would still equal the size of the original list.

Reply:
I don't like the fix; adding two whole new functions in a PR that's supposed to be simplifying things feels very not-worth-it.

Reply:
whoops, missed that you wanted to get rid of custody bitfield. 
Making the change, thanks!